blazor/datagrid/mySql.md

```diff
@@ -0,0 +1,588 @@
+---
+layout: post
+title: MySQL Data Binding in Blazor DataGrid Component | Syncfusion
+description: Learn about consuming data from MYSQL and binding it to Syncfusion Component, and performing CRUD operations
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# MySQL Data Binding in DataGrid
+
+This section describes how to use [MySQL data](https://www.nuget.org/packages/MySql.Data) to retrieve data from a MySQL server and bind it to the Blazor Pivot Table.
+
+## Connecting a MySQL database to a Syncfusion Blazor Grid
+
+**1.** Create a simple Blazor Grid by following the **"Getting Started"** documentation [link](https://blazor.syncfusion.com/documentation/datagrid/getting-started-with-web-app).
+
+**2.** To connect a MySQL using the MySQL driver in our application, we need to install the [MySQL.Data](https://www.nuget.org/packages/MySql.Data) NuGet package. To do so, open the NuGet package manager of the project solution, search for the package **MySQL.Data** and install it.
+
+![Add the NuGet package MySQL.Data to the project](./images/MySQL-nuget-package-install.png)
+
+**3.** Next, in the **Index.razor** page, get the MySQL data from the MySQL server and bind it to the DataGrid component as a datasource by using the Custom adaptor feature and configure the report to use the MySQL data.
```

Comment:
here datasource is mentioned as API same mention it in proper casing with API link